src/receipt.js

```diff
@@ -1,12 +1,14 @@
 import xml2js from 'xml2js';
 import TaxType from './tax-type';
 import Amount from './amount';
+import Utils from './common/utils';
 
 const defaultInvoiceAttr = {
-    xmlns: 'urn:GEINV:eInvoiceMessage:C0401:3.1',
-    'xmlns:xsi': 'http://www.w3.org/2001/XMLSchema-instance',
-    'xsi:schemaLocation': 'urn:GEINV:eInvoiceMessage:C0401:3.1 C0401.xsd'
-};
+        xmlns: 'urn:GEINV:eInvoiceMessage:C0401:3.1',
+        'xmlns:xsi': 'http://www.w3.org/2001/XMLSchema-instance',
+        'xsi:schemaLocation': 'urn:GEINV:eInvoiceMessage:C0401:3.1 C0401.xsd'
+    },
+    utils = new Utils();
```

Comment:
It is not necessary to make Utils to be a object.
If you export util with export default {...}, then importer can use them with util.FUNCTION().
https://developer.mozilla.org/zh-TW/docs/Web/JavaScript/Reference/Statements/import